include/gazebo_mavlink_interface.h

```diff
@@ -153,7 +153,7 @@ class GazeboMavlinkInterface : public ModelPlugin {
     groundtruth_lon_rad(0.0),
     groundtruth_altitude(0.0),
     mavlink_udp_port_(kDefaultMavlinkUdpPort),
-    qgc_udp_port_(kDefaultMavlinkUdpPort),
+    qgc_udp_port_(kDefaultQGCUdpPort),
```

Comment:
This changes it to port 14550. I wonder how the original version with `kDefaultMavlinkUdpPort` (14560) functioned correctly.

Reply:
Actually - do we ever use this QGC UDP link? Can we delete it completely? The PX4 executable should be routing the packets in reality.

Reply:
No. This is required because of HITL.